connectors/airwatch/src/main/service/rhel/files/default-conf/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+connector_name='airwatch'
```

Comment:
The airwatch start.sh is a little different than the others (it has the managed-apps.yml file as well as the 2 application.properties files).